xcodecache/swiftpm_cache.go

```diff
@@ -12,7 +12,8 @@ import (
 // SwiftPackagesStateInvalid is the partial error message printed out if swift packages cache is invalid.
 // Can be used to detect invalid state and clear the path returned by SwiftPackagesPath.
 // xcodebuild: error: Could not resolve package dependencies:
-//   The repository at [path] is invalid; try resetting package caches
+//
```

Comment:
Why do we need this change? It seems the last 2 lines of the comment is an actual error message printed by xcodebuild.

Reply:
I've just executed a gofmt command on the whole project. So technically It wasn't me. :D

Reply:
Can we revert this change, to keep the original format of this example error?